### PR TITLE
Fix the policyViolationsLimit field name in the policy wizard

### PR DESCRIPTION
**Pull request by mprahl**
Relates:
https://issues.redhat.com/browse/ACM-2946
https://github.com/open-cluster-management-io/governance-policy-propagator/pull/88

### CODE REVIEW

File: frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx [hunk 1]
```diff
@@ -175,8 +175,8 @@ export function PolicyAutomationWizard(props: {
                         hidden={(item) => !item.spec?.automationDef?.name}
                     />
                     <WizNumberInput
-                        path="spec.automationDef.policyViolationContextLimit"
-                        label={t('Policy Violation Context Limit')}
+                        path="spec.automationDef.policyViolationsLimit"
+                        label={t('Policy Violations Limit')}
```
Comment (KevinFCormier) [inline frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx:179]: @mprahl Are you OK if other languages have the wrong string until 2.7.1?

Comment (mprahl) [reply frontend/src/wizards/PolicyAutomation/PolicyAutomationWizard.tsx:179]: I'm okay with that. Thanks for pointing that out. I just updated the translation keys to match the new wording.

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**